Atomic/util.py

```diff
@@ -254,3 +253,19 @@ def urllib3_disable_warnings():
             # Except only call disable-warnings if it exists
             if hasattr(urllib3, 'disable_warnings'):
                 urllib3.disable_warnings()
+
+
+def skopeo(image):
```

Comment:
the behavior I'm telling about was _inside_ the docker remote inspection patch, but now that we removed it, we need to move this _loop through default registries_ logic into `atomic`